### PR TITLE
Log accuracy score in `EEGClassifier`

### LINKED ISSUE #510: Log accuracy score in `EEGClassifier`
Add the accuracy score to the callbacks of EEGClassifier. C.f.

```python
    @property
    def _default_callbacks(self):
         callbacks = super()._default_callbacks()
         if not self.cropped:
             callbacks.append((
                 'valid_acc', 
                 EpochScoring(
                     'accuracy',
                     name='valid_acc',
                     lower_is_better=False,
                 )
             ))
         return callbacks
```

_Originally posted by @PierreGtch in https://github.com/braindecode/braindecode/pull/501#discussion_r1317406658_

### LINKED ISSUE #521: Update default loss function
we will need to update the default loss function `EEGClassifier` to `nn.CrossEntropyLoss` because of #513  
Also, we will need to change the methods `predict`, `predict_probas`, and `predict_log_probas` .
Finally, we need to document that `EEGClassifier` expects models that don’t have a softmax/logsoftmax layer.

### PR DESCRIPTION
**Pull request by PierreGtch**
closes #510 closes #521

### CODE REVIEW

**Review by bruAristimunha — APPROVED** [review]

**Comment by bruAristimunha** [comment]
is there somewhere missing?

**Comment by PierreGtch** [comment]
@bruAristimunha all good to merge

**Review by bruAristimunha — APPROVED** [review]

**Comment by bruAristimunha** [comment]
Thank you @PierreGtch =)

**Comment by robintibor** [comment]
I noticed the default valid accuracy callback now causes me quite some issues in writing examples, things are logged twice etc. Was there a specific reason this was needed? @PierreGtch  Considering to undo this part (not the cross entropy loss default).

**Comment by PierreGtch** [comment]
@robintibor I just thought it made sense to log the accuracy by default for classifiers.
If it helps, see how I deactivated it in the `plot_sleep_staging_usleep.py` example: https://github.com/braindecode/braindecode/pull/541/files#diff-6fce3d4fb0b5aed7465d731b19bbf3ff2a3016d32e1d6e58d006fb5bf67a3b74 
Otherwise, if it creates too many issues, we can remove it.
